### DISCUSSION TITLE
New Vercel account shows blocked instead of registration

### DISCUSSION THREAD

**Discussion by tritechsolutionsltd** [vercel/community#3174] [Help] [answered]
### Summary

I've created a new GitHub account and tried to signup to Vercel using my GitHub account but after authorizing from GitHub Vercel says my account is blocked. How is it possible I've not even logged in once into my dashboard.

### Steps to Reproduce

Create a new GitHub account and signup on Vercel using GitHub.

**Comment by amyegan** [comment, marked as answer]
Please email [registration@vercel.com](mailto:registration@vercel.com) and include the  email, phone number, and username used for account creation. To protect against platform abuse, Vercel employs multiple layers of security and verification. Providing additional information about your account and the project you are deploying can help accelerate this process.
